controllers/grid/settings/sections/form/SectionForm.inc.php

```diff
@@ -15,10 +15,13 @@
 
 import('lib.pkp.controllers.grid.settings.sections.form.PKPSectionForm');
 
-use \APP\template\TemplateManager;
+use APP\template\TemplateManager;
 
 class SectionForm extends PKPSectionForm
```

Comment:
Could we use the existing section setting "Will not be included in the indexing of the journal"?

Reply:
I'm not sure what that's used for, but I think we should have a distinct setting. That's how the "browseBySection" plugin handles it and I think it's good to be configurable. I might, for example, only want some sections to appear in a list of browsable sections because I've "retired" others.

Reply:
Actually, on second thought, I think that we now have the "inactive" setting for sections. So maybe we can just use that instead... I'll need to look into how that works though.

Reply:
But having a section inactive means that you can not submit there right? That would not work with the example in your first message.

I checked and metaIndexed is dead code, it does not affect anything in OJS or OPS. What if we change the labeling a bit and use that?

Reply:
check